.github/workflows/pr-reject-png-jpg.yml

```diff
@@ -0,0 +1,32 @@
+name: Reject JPEG/PNG files
+
+on:
+  pull_request:
+    branches: [main]
+
+jobs:
+  reject-jpg-png:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0  # Fetch all history to ensure we can access all branches
+
+      - name: Fetch main branch
```

Comment:
Just curious, is this step necessary? I see the previous step already fetches the data.

Reply:
You are completely right! Removed this step here and in `pr-check-file-names`.